apps/mull-ui/src/app/components/custom-file-upload/custom-file-upload.tsx

```diff
@@ -19,11 +20,17 @@ export const CustomFileUpload = ({
   errorMessage,
   handleFileUpload,
   fieldName,
+  isEditProfile,
 }: CustomFileUploadProps) => {
   return (
     <div className="custom-file-upload-container">
       <label htmlFor={fieldName} className={`${imageURL ? 'image-uploaded' : ''}`}>
-        {imageURL ? (
+        {imageURL && isEditProfile ? (
+          <>
+            <img className="custom-file-upload-image" src={imageURL} alt="Event" />
+            <FontAwesomeIcon className="pencil-icon" icon={faPencilAlt} />
+          </>
+        ) : imageURL ? (
           <img className="custom-file-upload-image" src={imageURL} alt="Event" />
         ) : (
           <div className={className}>
```

Comment:
Having chained ternary operators is confusing especially when they're this long. We extract the conditional rendering of the pencil icon into its own conditional block like below.
```
{imageURL ? (
  <img className="custom-file-upload-image" src={imageURL} alt="Event" />
) : (
  <div className={className}>
    <FontAwesomeIcon className="custom-file-upload-icon" icon={faImages} />
  </div>
)}
{isEditProfile && <FontAwesomeIcon className="pencil-icon" icon={faPencilAlt} />}
```

Reply:
Done.